CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 -----------
 
 ### Internals
-* None.
+* Improved performance on `find_first` for small (< 64 byte) string arrays.
```

Comment:
Small strings are <= 15

Reply:
Ah you're right for string columns and that's probably the most common case. I was thinking about finding a table by it's name; table names are allowed to be up to 64 characters and are stored in an ArrayString.

Reply:
Maybe we should just omit the size thing in the change description.  Instead we could stress that this will improved the table name lookup performance.

Reply:
@ironage I have commited yet another description. Make a better one if you wish to. I approve whatever you come up with :-)
Squash the commit on merge - we don't want all the interim commits to show up..